apple/swift-docc-render
# Allow non-menu-closing tray links

Bug/issue #, if applicable: 92984703

## Summary

Allows for specifying a "noclose" class on links to prevent the default mobile behavior of closing the nav after it is clicked.

## Testing

Steps:
1. Add the "noclose" CSS class to link in the nav menu tray.
2. Verify that clicking the link no longer triggers the menu tray to close before the link action takes place.

## Checklist

Make sure you check off the following items. If they cannot be completed, provide a reason.
